gpfdist-sink/src/main/java/org/springframework/cloud/stream/module/gpfdist/sink/GpfdistServer.java

```diff
@@ -148,7 +148,7 @@ public Buffer apply(Buffer prev, Buffer next) {
 				request.responseHeaders().removeTransferEncodingChunked();
 				request.addResponseHeader("Content-type", "text/plain");
 				request.addResponseHeader("Expires", "0");
-				request.addResponseHeader("X-GPFDIST-VERSION", "Spring XD");
+				request.addResponseHeader("X-GPFDIST-VERSION", "Spring Dataflow");
```

Comment:
Doesn't have to be dataflow specific, as this will eventually go into the app-starters repo, may call it Spring GPFDIST Stream App.   Why is the version not a number?  Just cosmetics I realize....